tests/sync/initialSync.js

```diff
@@ -0,0 +1,48 @@
+const { assert } = require('chai');
+const util = require('./util');
+const helperDb = require('../helperDb');
+
+describe('Initial Sync', () => {
+  beforeEach('set up testing database', helperDb.before);
```

Comment:
I think this needs to now be `before()` after Tim’s changes were merged.

Reply:
I think that's only for `helperServer` - we don't need a full server here, so I'm just using `helperDb`.

Reply:
Right!